src/server.rs

```diff
@@ -83,22 +82,87 @@ impl<'f> FromForm<'f> for Options {
     }
 }
 
+impl Options {
+    pub fn from_rawstr(string: &str) -> Result<Options, HandlerError> {
```

Comment:
Since this totally replaces the `impl FromForm for Options` let's remove that impl entirely